docs/source/index.rst

```diff
@@ -1,3 +1,21 @@
+..
+ Copyright 2022 Graviti. Licensed under MIT License.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: GETTING STARTED
+
+   getting_started/installation
+   getting_started/10_minutes_to_graviti
+
+.. toctree::
+   :maxdepth: 1
+   :caption: FEATURES
+
+   features/dataset_management/index
+   features/version_control/index
+   features/sheet_management/index
```

Comment:
Shall we move the `sheet_mamagement` under the `dataset_management`? 
It might look more ordered.